src/app/public/modules/progress-indicator/progress-indicator.component.ts

```diff
@@ -216,4 +260,62 @@ export class SkyProgressIndicatorComponent implements AfterContentInit, OnDestro
       return (i === index);
     });
   }
+
+  private handleIncomingMessage(
+    message: SkyProgressIndicatorMessage | SkyProgressIndicatorMessageType
+  ): void {
+    const value: any = message;
+
+    let type: SkyProgressIndicatorMessageType;
+
+    // Prints a deprecation warning if the consumer provides only `SkyProgressIndicatorMessageType`.
+    if (value.type === undefined) {
+      console.warn(
+        '[Deprecation warning] The progress indicator component\'s `messageStream` input was ' +
+        'set to `Subject<SkyProgressIndicatorMessageType>`. This is a ' +
+        'deprecated type and will be removed in the next major version release. ' +
+        'Instead, set the `messageStream` input to `Subject<SkyProgressIndicatorMessage>`.'
+      );
```

Comment:
From @blackbaud-johnly - we should be writing these in the active voice. For example:

[Deprecation warning] The progress indicator component's `messageStream` input is set to `Subject<SkyProgressIndicatorMessageType>`. We will remove this deprecated type in the next major version release. Instead, set the `messageStream` input to `Subject<SkyProgressIndicatorMessage>`.

Reply:
Good catch.